Cargo.toml

```diff
@@ -24,7 +24,7 @@ version = "0.16.5"
 features = ["extension-module", "hashbrown", "num-complex", "num-bigint"]
 
 [dependencies.ndarray]
-version = "^0.15.0"
+version = "^0.15.6"
```

Comment:
Let me beck the syntax on this, I don't know if we really need to update this. It doesn't hurt but I would have assumed the `^` prefix would match any patch version

Reply:
Heh, looks like it doesn't mean anything special besides the default behavior https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#caret-requirements